src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Services/UriHelper.ts

```diff
@@ -22,7 +22,10 @@ registerFunction(`${registeredFunctionPrefix}.enableNavigationInterception`, ()
     const anchorTarget = findClosestAncestor(event.target as Element | null, 'A');
     if (anchorTarget) {
       const href = anchorTarget.getAttribute('href');
-      if (isWithinBaseUriSpace(toAbsoluteUri(href))) {
+        console.log(event);
+      if (isWithinBaseUriSpace(toAbsoluteUri(href)) && 
+          !event.ctrlKey) 
```

Comment:
I think it's not only ctrl that needs to be accounted for. There's also shift and meta, depending on the browser OS. Would you be able to determine the desired behavior for each such modifier key?

Reply:
ok I'll try to find more details on this